src/Option.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Option handler
+ *
+ * @package Yoast\Test_Helper
+ */
+
+namespace Yoast\Test_Helper;
+
+/**
+ * Class to manage registering and rendering the admin page in WordPress.
+ */
+class Option {
+	/**
+	 * The name of our option.
+	 *
+	 * @var string
+	 */
+	public static $option_name = 'yoast_test_helper';
```

Comment:
Passing this variable via a constructor would make it much more useful in a generic sense.
There is no logic specifically for the configured option.

All methods can just be `non-static` methods.

The classes that use the option can hold the instance of the option as class-variable, providing the static-ness nature of the current setup.